mappings/net/minecraft/entity/passive/CamelEntity.mapping

```diff
@@ -0,0 +1,43 @@
+CLASS net/minecraft/unmapped/C_lzclwwts net/minecraft/entity/passive/CamelEntity
+	FIELD f_chodqqna DASH_VERTICAL_MOMENTUM F
+	FIELD f_gxjuclwy RUNNING_SPEED_BONUS F
+	FIELD f_jnrgxjnd DIMENSIONS Lnet/minecraft/unmapped/C_sszpscpo;
```

Comment:
What do we use for other entities?

Reply:
Seems to be no exact standard, I've come across `dimensions` the most though